core/src/main/java/com/linecorp/armeria/server/DefaultUnhandledExceptionsReporter.java

```diff
@@ -106,6 +109,7 @@ private void reportException() {
                         "detailed error logs. One of the thrown exceptions:",
                         newExceptionsCount,
                         TextFormatter.elapsed(intervalMillis, TimeUnit.MILLISECONDS), exception);
+            thrownExceptionCounters.computeIfAbsent(exception, k -> new LongAdder()).increment();
```

Comment:
I've implemented recording the exception in the `thrownExceptionCounters` when it's not null. 
Did I understand the intention correctly? 
Please let me know if I've misunderstood anything. 
Thank you. 🙇